README.md

```diff
@@ -0,0 +1,701 @@
+# Capability based Access Control in `p2panda` [under review]
+
+# Introduction
+
+This document describes an access control system for the [p2panda protocol](https://p2panda.org) which gives peers fine grained control over who can read their data, and what actions they are authorized to perform on it. The system must take account of the particular challenges faced as a result of supporting offline-first functionality and delay tolerant networking.
+
+# Background
+
+Peer-to-peer protocols such as p2panda use public-key cryptography and digital signatures to establish the identity of peers and the authenticity of messages replicated on a network. Any peer can verify that messages it receives were created by the claimed peer (public key), and that they have not been tampered with by any third parties. This allows peers to replicate messages freely, safe in the knowledge that if tampering occurs it can be detected.
+
+What if we don't want to share data with anyone though? Even if we know it won't be tampered with, we may want a stricter system where an author has additional control over where their data can travel, and how peers are allowed to interact with it. If we understand "authorship" as equal to "ownership" then we can say that only an owner has "read" and "write" authority over the data they created ("write" authority being relevant to long-lived mutable data).
```

Comment:
I think the word "owner" loads dangerous expectations in a distrbuted digital space. Namely "i will now delete *my* record". Good luck, "owner"

Reply:
Yep, I appreciate your point for sure. I'll argue the point little, but also consider other options. 

To me it works here as introducing the ability to act as owner is exactly what we are trying to do here with access control and capabilities. With this system the default behaviour is for the creator of a resource (now called owner) to not share it with anyone unless they explicitly give permission (hand out a capability). They own the right to decide how they distribute their resources, they can share it widely or only with some very close friends, and they can specify (via delegation) how others can interact with their resource. Of course the more you do this, the less control you have in malicious scenarios. 

In terms of how we understand ownership in other areas of our lives, this seems quite aligned. I own a microphone, I share it with you, I'm still the owner although you _have_ the microphone sometimes. (I'm only talking about understanding of the word here, not the actual mechanics of the system).